packages/features/bookings/Booker/utils/event.ts

```diff
@@ -19,7 +19,7 @@ export type useScheduleForEventReturnType = ReturnType<typeof useScheduleForEven
  * Using this hook means you only need to use one hook, instead
  * of combining multiple conditional hooks.
  */
-export const useEvent = (props?: { fromRedirectOfNonOrgLink?: boolean }) => {
+export const useEvent = (props?: { fromRedirectOfNonOrgLink?: boolean; currentUserId?: number }) => {
```

Comment:
This would make `useEvent` dependent on `useSession` and we should get the logged in user id on the server